test/Microsoft.Extensions.CodeGeneration.Core.FunctionalTest/TestHelper.cs

```diff
@@ -30,8 +30,10 @@ public static IServiceProvider CreateServices(string testAppName)
             var originalAppBase = appEnvironment.ApplicationBasePath; ////Microsoft.Extensions.CodeGeneration.Core.FunctionalTest
             var testAppPath = Path.GetFullPath(Path.Combine(originalAppBase, "..", "TestApps", testAppName));
             var testEnvironment = new TestApplicationEnvironment(appEnvironment, testAppPath, testAppName);
+            var rid = Microsoft.Extensions.PlatformAbstractions.RuntimeEnvironmentExtensions.GetRuntimeIdentifier(
```

Comment:
Scaffolding has it's own `IApplicationEnvironment` so I need to specify full namespaces.